### PR TITLE
Create mikelxc.json

### LINKED ISSUE #3153: Please add me - Suzhou, China
Hello :wave:! Please add me as a collaborator to this repo.

### PR DESCRIPTION
**Pull request by mikelxc**
@githubteacher
**Please** help me pin my city!!!!
closes #3153

### CODE REVIEW

**Comment by mikelxc** [comment]
@githubteacher how to solve this problem?

**Comment by mikelxc** [comment]
@crichID Can you help me with this problem?

File: _pin/mikelxc.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+ ---
```
Comment (crichID) [inline _pin/mikelxc.json]: @mikelxc you will need to remove the leading spaces from each line of the file!

**Comment by mikelxc** [comment]
@crichID Is that ok?

**Comment by mikelxc** [comment]
@crichID Thank you! The problem is solved!

**Comment by crichID** [comment]
@mikelxc :sparkles: great work!

**Pull request merged**